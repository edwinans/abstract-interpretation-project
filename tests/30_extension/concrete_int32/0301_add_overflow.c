{
  int x;
  int y;
  y = 1024; 
  x = y * y * y + 5; // 2^30 + 5
  x = x + x; // overflow > 2^31 - 1
  print(x);
}
