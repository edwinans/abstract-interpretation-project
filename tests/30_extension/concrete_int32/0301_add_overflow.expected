tests/30_extension/concrete_int32/0301_add_overflow.c:7.2-11: { [x=-2147483638;]; }
analysis ended
